### ISSUE TITLE
At least the home page is failing WAVE, perhaps others

### ISSUE THREAD

**Issue by mathieuhendey** [shylands/climatechoice#121]
We have quite a few WAVE errors on at least the homepage. We should be running WAVE on all new screens, and I  recommend that we doublecheck this before merging again. Or set up CI/CD

[
<img width="215" alt="image" src="https://user-images.githubusercontent.com/1131786/59146925-5271fd00-89ec-11e9-92f7-cce39b2c70c6.png">
](url)

**Comment by eostrom** [comment]
I can't get WAVE to work on the home page at all, in Chrome or Firefox. It puts up a loading spinner, logs some console errors, and hangs indefinitely.

Bright side, it gets 100% on Lighthouse's accessibility audit.

**Comment by Cazaimi** [comment]
@mathieuhendey , may I take up this issue?

**Comment by heatherpekrul** [comment]
WAVE is a good extension, but the [Axe Coconut extension](https://chrome.google.com/webstore/detail/axe-coconut-web-accessibi/iobddmbdndbbbfjopjdgadphaoihpojp) is more robust and includes more details, and occasionally catches issues that WAVE does not. I'd recommend it for accessibility testing over WAVE.

**Comment by andrew-pyle** [comment]
I just submitted a PR that adds [eslint-plugin-jsx-a11y](https://www.npmjs.com/package/eslint-plugin-jsx-a11y) checking to git pre-commit hooks. This is similar to WAVE, and easy to enforce.
